modules/web_hosting/sql.tf

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+ resource "google_sql_database_instance" "db_postgres" {
```

Comment:
Replace with https://github.com/terraform-google-modules/terraform-google-sql-db

Reply:
I have variablized the google_sql_database_instance resource. I am finding https://github.com/terraform-google-modules/terraform-google-sql-db with limited options and less documentation where https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/sql_database_instance has complete details.